doc/code_snippets/snippets/replication/instances.enabled/manual_leader/config.yaml

```diff
@@ -3,9 +3,6 @@ credentials:
     replicator:
       password: 'topsecret'
       roles: [replication]
-    client:
-      password: 'secret'
-      roles: [super]
```

Comment:
I'm in doubts here. We have a replicaset and nobody can connect to it (except using an admin console).

Reply:
Yep, I understand your doubts but I'd like that replication samples demonstrate only options related to replication to make them more concise. For credentials options, I plan to create a separate sample that will demonstrate all use cases related to connecting, executing functions, and so on. 

https://docs.d.tarantool.io/en/doc/3.0-config-replication/reference/configuration/configuration_reference/#credentials